Themis/Views/Assessment/AssessmentView.swift

```diff
@@ -254,13 +258,16 @@ struct AssessmentView: View {
             }
         }
         .sheet(isPresented: $cvm.showAddFeedback) {
+            cvm.selectedFeedbackSuggestionId = ""
```

Comment:
I don't know if that's usual SwiftUI knowledge, however personally I didn't know off the top of my head that this function was for `onDismiss`. Could we use the different syntax where you include the label for this function?